doc/source/how-to/documenting.rst

```diff
@@ -469,349 +470,115 @@ branch within this repository. This is done by uploading the generated
-        Running post deployment cleanup jobs… 🗑️
-        /usr/bin/git checkout -B github-pages-deploy-action/x00pqaqlu
-        Reset branch 'github-pages-deploy-action/x00pqaqlu'
-        /usr/bin/chmod -R 777 github-pages-deploy-action-temp-deployment-folder
-        /usr/bin/git worktree remove github-pages-deploy-action-temp-deployment-folder --force
-        Error: The deploy step encountered an error: The process '/usr/bin/git' failed with exit code 128 ❌
-
-
-Paste the value of the token in the ``Settings/Secrets/Actions`` path under a
-new secret named ``DEPLOY_DOCS_PAT`` in the repository of the project. Use this
-secret in your CI/CD:
-
-.. tab-set::
-
-    .. tab-item:: Using ``tox``
-
-        .. code-block:: yaml
-
-            - name: Deploy
-              if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
-              uses: JamesIves/github-pages-deploy-action@v4.3.0
-              with:
-                token: ${{ secrets.DEPLOY_DOCS_PAT }}
-                repository-name: pyansys/repository-name
-                branch: gh-pages
-                folder: .tox/doc_out
-                clean: true
-
-    .. tab-item:: Without using ``tox``
-
-        .. code-block:: yaml
-
-            - name: Deploy
-              if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
-              uses: JamesIves/github-pages-deploy-action@v4.3.0
-              with:
-                token: ${{ secrets.DEPLOY_DOCS_PAT }}
-                repository-name: pyansys/repository-name
-                branch: gh-pages
-                folder: doc/_build/html
-                clean: true
+Deploying to ``gh-pages`` of the repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+While you could manually upload your auto-generated documentation for each
+release using your own local GitHub credentials, the best practice is to have
+your documentation build on GitHub and deployed either on release or push to
+main. You can do this via `GitHub Actions`_ by taking advantage of
+``pyansys/actions`` reusable workflows.
 
+Generated documentation gets published in the ``gh-pages`` branch of the
+repository of the project. On top of that, two artifacts named
+``documentation-html`` and ``documentation-pdf`` get generated at the end of the
+workflow.
 
+Add the following workflow job to build and deploy development and stable
+documentation in an automated way.
 
-Deploy when tagging
-+++++++++++++++++++
+.. code-block:: yaml
 
-Depending on your preferences, you can choose to update the documentation on
-tags only (as done earlier) or on each push. If you want to have your
-documentation deployed on each push to ``main``, change the preceding conditional
-to:
+    doc-build:
+      name: "Building documentation"
+      runs-on: ubuntu-latest
+      steps:
+        - name: "Run Ansys documentation building action"
+          uses: pyansys/actions/doc-build@v1
```

Comment:
Here you are pointing at ``@v1``  while in the ``yml`` files you point at ``@main``. I do think it's better to point at a stable version rather than main for these workflows. Let dependabot update them if they ever copypaste from here =)

This will imply changing all ``yml`` files.

Reply:
Sure, let me make sure that everything points to the `@v1`. This may be a result of copy-pasting from the development documentation of the actions.

Reply:
Solved!